src/repl.zig

```diff
@@ -1904,3 +1919,40 @@ test "repl.zig: Handle parsing errors" {
         try std.testing.expectError(t.err, result);
     }
 }
+
+test "repl.zig: Parser fails for operations not supporting multiple objects" {
+    const tests = [_]struct {
+        in: []const u8,
+        err: anyerror,
+    }{
+        .{
+            .in = "get_account_transfers account_id=1, account_id=2",
+            .err = error.UnsupportedSliceOperation,
+        },
+        .{
+            .in = "get_account_balances account_id=1, account_id=2",
+            .err = error.UnsupportedSliceOperation,
+        },
+        .{
+            .in = "query_accounts account_id=1, account_id=2",
+            .err = error.UnsupportedSliceOperation,
+        },
+        .{
+            .in = "query_transfers account_id=1, account_id=2",
+            .err = error.UnsupportedSliceOperation,
+        },
+    };
+
+    for (tests) |t| {
```

Comment:
```suggestion
    for (vectors) |vector| {
```

Per TigerStyle, don't abbreviate variable names.

Also a practice of the code base to try to use names that have a clear singular and plural. And these are more test vectors (within a test).